bench/chromium/corpus_serve.py

```
@@ -149,18 +158,54 @@ def send_header(self, keyword, value):
+        Origin rather than "*": credentialed fetches (CMP/geo APIs) reject
+        the wildcard, and those gate consent overlays the pixel check sees.
+
+        A miss needs them as much as a hit does. Without them the browser
+        blocks the 404 before the renderer sees it, reports net::ERR_FAILED,
+        and receives no response, so the trace records no remote address and
+        cannot say the request was answered here rather than off the box.
+        Measured 2026-08-29 over a 42-render diag: 32 traced rows carried no
+        remote address, every one of them cross-origin and every one answered
+        by this server (27 GET 404, 2 POST 404, 3 HEAD 501). The status is
+        unchanged, so a url the corpus does not hold still says so, in the
+        status, in the access log and in /––misses.
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)